functions/notify_slack.py

```diff
@@ -15,6 +15,15 @@ def decrypt(encrypted_url):
   except Exception:
     logging.exception("Failed to decrypt URL with KMS")
 
+def get_ssm_param(ssm_param):
+  try:
+    ssm = boto3.client("ssm")
```

Comment:
i would move the client init outside of the function to cache it and open connection each time :D